### PR TITLE
Fix banner external route option

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request?
FIx banner external route option

#### What problem is this solving?
The banner external route was not been showing on the storefront as an option.

#### How should this be manually tested?
[Access the storefront](https://theoo--storecomponents.myvtex.com/admin/cms/storefront) and go to carousel section.

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/Carousel.tsx [hunk 1]
```diff
@@ -287,21 +283,24 @@ export default class Carousel extends Component<Props, State> {
...
-                <Banner height={height} {...banner} />
-              </Slide>
-            )
-          )}
+          {banners.map((banner, i) => (
+            <Slide
+              className={styles.slide}
+              key={i}
+              style={{ maxHeight: height }}
+              sliderTransitionDuration={500}
```
Comment (matheusps) [inline react/Carousel.tsx]: Shouldn't it be a defaultProp? 🤔

Comment (Klynger) [reply react/Carousel.tsx]: This is not actually a prop, but it could be a new thing in the schema I think.

Comment (theomoura) [reply react/Carousel.tsx]: I think we should keep this variable in an global variable

Comment (matheusps) [reply react/Carousel.tsx]: Got it

**Pull request merged**